spring-cloud-sleuth-brave/src/main/java/org/springframework/cloud/sleuth/brave/autoconfig/BraveBaggageConfiguration.java

```diff
@@ -64,10 +63,9 @@
  * @since 2.0.0
  */
 @Configuration(proxyBeanMethods = false)
-@EnableConfigurationProperties(SleuthBaggageProperties.class)
-class TraceBaggageConfiguration {
+class BraveBaggageConfiguration {
```

Comment:
I think the naming could use some polishing. Stuff start with `Trace` others with `Brave` and yet others with `TraceBrave`.

Reply:
Not related to this PR but GitHub won't let me add a comment on a unchanged line. You can't put a `ConditionalOnClass` on a method. In this particular case it works because the type is not part of the signature but's a fragile arrangement (see `correlationScopeDecoratorBuilder`).